plugins/dev-inspector/dev-inspector.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Plugin Name:       Dev Inspector
+ * Description:       Example block scaffolded with Create Block tool.
+ * Version:           0.1.0
+ * Requires at least: 6.7
+ * Requires PHP:      7.4
+ * Author:            The WordPress Contributors
+ * License:           GPL-2.0-or-later
+ * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
+ * Text Domain:       dev-inspector
+ *
+ * @package DeveloperShowcase
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+/**
+ * Registers the block using a `blocks-manifest.php` file, which improves the performance of block type registration.
+ * Behind the scenes, it also registers all assets so they can be enqueued
+ * through the block editor in the corresponding context.
+ *
+ * @see https://make.wordpress.org/core/2025/03/13/more-efficient-block-type-registration-in-6-8/
+ * @see https://make.wordpress.org/core/2024/10/17/new-block-type-registration-apis-to-improve-performance-in-wordpress-6-7/
+ */
+function developer_showcase_dev_inspector_block_init() {
```

Comment:
I think we should just assume that we're running 6.8+ and can remove all the extra function_exists checks.